src/DB/projects.json

```diff
@@ -2333,5 +2333,13 @@
         "tech": ["HTML", "Tailwind CSS"]
       }
     ]
+  },
+  {
+    "github_username": "sditya-stoic-sharma",
+    "Social_media": {
+      "gitHub": "https://github.com/aditya-stoic-sharma",
+      "LinkedIn": ""
+    }
```

Comment:
hey please create a another PR for projects addition

and remove the code from here

Reply:
ohkkk @priyankarpal